sql/mob_spawn_points.sql

```diff
@@ -33183,7 +33183,7 @@ INSERT INTO `mob_spawn_points` VALUES (17420449,'Goblin_Shaman','Goblin Shaman',
 INSERT INTO `mob_spawn_points` VALUES (17420450,'Goblin_Pathfinder','Goblin Pathfinder',8358,-450.284,-95.529,-35.169,114);
 INSERT INTO `mob_spawn_points` VALUES (17420451,'Goblin_s_Bat','Goblin\'s Bat',8361,1,1,1,0);
 INSERT INTO `mob_spawn_points` VALUES (17420452,'Goblin_Furrier','Goblin Furrier',8357,0,0,0,0);
-INSERT INTO `mob_spawn_points` VALUES (17420453,'Goblin_Smithy','Goblin Smithy',8360,-482.261,-95.529,-20.183,48);
+-- INSERT INTO `mob_spawn_points` VALUES (17420453,'Goblin_Smithy','Goblin Smithy',8360,-482.261,-95.529,-20.183,48);
 INSERT INTO `mob_spawn_points` VALUES (17420454,'Goblin_Shaman','Goblin Shaman',8359,-33.756,-80.045,-46.854,127);
```

Comment:
@Loki79 these 2 goblins, as well as a weapon in shrine of ru'avawhatever, I commented out to prevent clients dc'ing from large packet after teleport. I dunno if dsp wants them commented out tho (I would wager they don't).